contrib/hostapd/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "hostapd"
+pkgver = "2.10"
+pkgrel = 0
+build_wrksrc = "hostapd"
+build_style = "makefile"
+make_cmd = "gmake"
+make_install_args = ["BINDIR=/usr/bin"]
+make_use_env = True
+hostmakedepends = ["pkgconf"]
+makedepends = ["gmake", "libnl-devel", "linux-headers", "openssl-devel"]
+pkgdesc = "IEEE 802.11 AP, IEEE 802.1X/WPA/WPA2/EAP/RADIUS Authenticator"
+maintainer = "Renato Botelho do Couto <renato@netgate.com>"
+license = "BSD-3-Clause"
+url = "https://w1.fi/hostapd"
+source = f"https://w1.fi/releases/{pkgname}-{pkgver}.tar.gz"
```

Comment:
```suggestion
source = f"https://w1.fi/releases/hostapd-{pkgver}.tar.gz"
```

Reply:
Out of curiosity, what is the reason to not use pkgname on source URLs?

Reply:
we just decided to not do that anymore at some point. it's not really a variable since if the template is copied and the pkgname changed the source becomes wrong, and in cases where you have like github/project/{pkgname} you lose the ability to copy the url quickly